sql/core_clients_daily_v1.sql

```diff
@@ -0,0 +1,76 @@
+WITH
+  numbered_duplicates AS (
+  SELECT
+    * REPLACE(LOWER(client_id AS client_id)),
+    ROW_NUMBER() OVER (PARTITION BY client_id, submission_date_s3, metadata.document_id ORDER BY metadata.timestamp DESC) AS n
+  FROM
+    telemetry_core_parquet_v3
+  WHERE
+    client_id IS NOT NULL ),
+  -- Deduplicating on document_id is necessary to get valid SUM values.
+  deduplicated AS (
+  SELECT
+    * EXCEPT (n)
+  FROM
+    numbered_duplicates
+  WHERE
+    n = 1 ),
+  windowed AS (
+  SELECT
+    @submission_date AS submission_date,
+    CURRENT_DATETIME() AS generated_time,
+    client_id,
+    ROW_NUMBER() OVER w1 AS n,
+    -- For now, we're ignoring the following RECORD type fields:
+    --   accessibility_services
+    --   experiments
+    --   searches
+    --
+    -- Take the min observed profile creation date.
+    SAFE.DATE_FROM_UNIX_DATE(MIN(profile_date) OVER w1) AS profile_date,
+    -- These integer fields are already sums over sessions since last upload,
+    -- so we sum to represent all uploads in the given day;
+    -- we set some upper limits to avoid integer overflow on pathological input.
+    SUM(LEAST(sessions, 1000000)) OVER w1 AS sessions,
+    SUM(LEAST(durations, 1000000)) OVER w1 AS durations,
+    SUM(LEAST(flash_usage, 1000000)) OVER w1 AS flash_usage,
```

Comment:
Somewhat arbitrary choice of 1 million as the max value for these integers in any given ping. 1 million seems erroneously high either for a count of uses or for a count of seconds. I was seeing integer overflows in practice without imposing this limit.

Reply:
Excluding clearly-bogus values here seems reasonable, but as an alternate strategy here's a query I prototyped that includes 99.9% of the values and excludes the highest 0.1% as outliers:

https://sql.telemetry.mozilla.org/queries/61050/source#157406

Also does the ping schema already ensure these values are `>= 0`?

Reply:
> Also does the ping schema already ensure these values are >= 0?

There are indeed values less than zero. Will think about fixing that.

Also, p999 is a good idea. I just took a look and found that value to be ~28000 for durations, so perhaps 100,000 is a good cutoff; will document.